project_app/components/Footer.js

```diff
@@ -7,17 +7,38 @@ import {
   ProductContainer,
 } from "./styled";
 import Image from "next/image";
+import Link from "next/link";
 
 export default function Footer() {
-  const titles = ["About Us", "FAQ", "Contact Us", "Give Feedback"];
-
+  const titles = [
+    {
+      name: "About",
+      link: "/about",
+    },
+    {
+      name: "FAQ",
+      link: "/faq",
+    },
+    {
+      name: "Contact Us",
+      link: "/contact",
+    },
+    {
+      name: "Give Feedback",
+      link: "/feedback",
```

Comment:
in that case, let's make the page files lowercase

Reply:
Done